test/browser/content-types/add-content-type.spec.ts

```diff
@@ -83,4 +84,13 @@ describe('Add content type', () => {
             }
         });
     });
+
+    it(`text type element should contain validation regex`, () =>
+    {
+        const contentType = response.data.elements.find(m => m.type === 'text');
+        expect(Object.keys(contentType!)).toContain('validation_regex');
```

Comment:
@IvanKiral Wouldn't it be better if you compared actual values from "raw" json and the one you get from delivery response? ;) This seems to be only checking for existence of the props, but not their values.

Reply:
I was thinking that the test above this one, which tests whether the values are mapped are checking if it compared, because of that I thought that I only check the pressence. But if that is not correct I might change it :)

Reply:
In most tests I have I try to compare values from the fake json to what is received by management client - just to make sure that sdk didn't tampered with the data or anything ;) If you can compare values, that would be great!

Reply:
Okay :) I pushed a new version, hopefully I got this right :D